unicode_names2_macros/Cargo.toml

```
@@ -18,8 +18,8 @@ description = "Support macros for `unicode_names2`."
 unstable = []
 
 [dependencies]
-regex = "0.1.80"
-syn = "0.15"
+regex = "1.9.3"
+syn = "2.0.28"
```

```suggestion
regex = "1.0"
syn = "2.0"
```

We can relax the requirements without breaking the build.